CHANGELOG.md

```diff
@@ -0,0 +1,47 @@
+# Mattermost Mobile Apps Changelog
+
+## Beta Release
+
+- Planned Release Date: End of March, 2017
+- Server Versions Supported: Server v3.7+ is required
+
+Development for the Beta Release is currently in progress, features implemented so far are listed below. 
+
+#### Authentication
+- Email login
+- LDAP/AD login
+- Multi-factor authentication 
+- Logout
+
+#### Messaging
+- View and send posts in the center channel
+- View and send replies in thread view
+- "New messages" line in center channel (app does not yet scroll to the line)
+- @mention autocomplete
+- "User is typing" message
+
+#### User profiles
+- Status indicators
+- View profile information by clicking on someone's username or profile picture
+
+#### Files
+- Generic thumbnails when files are attached to a post (cannot open the file to view it yet) 
+
+#### Channels
+- Channel drawer for selecting channels
+- Bolded channel names for Unreads, and mention jewel for Mentions
+- Favorite channels
```

Comment:
It might not be clear whether this means the ability to favorite/unfavorite channels or just that favorite channels are grouped separately in the channels drawer.